### PR TITLE
Added the ability to download every available video.

### PR DESCRIPTION
**Pull request by Tantalum73**
I added a button in the settings window that lets the user download every video (WWDC 2015) that have been released yet.

The button will open the downloads-window and start downloading the first video immediately (only once at a time). It does so by skipping every video, that is already on disk.

If you are like me, you want every video on your disk to watch it offline or at 2x speed.
Therefore, you should not klick "download" on every single available video.

### CODE REVIEW

**Comment by insidegui** [comment]
For some reason this crashes in El Capitan. Not only when I click "Download all videos", but whenever I click to download a video. It wasn't crashing before. Can you debug It?

**Comment by Tantalum73** [comment]
Phew I don't have the captain installed yet.
Do you know which line is causing the crash?

**Comment by insidegui** [comment]
It's crashing in `VideoStore` line `67`, and there's an error about the session being invalid or something. 

Sorry, I switched back to my Yosemite partition to see if the bug appeared here and forgot to take screenshot/copy the log messages ><

**Comment by Tantalum73** [comment]
That is strange, due to the fact that I only call download(url: String) for every session. Just like a user initiated click would do, but starting the next video when one is finished.

**Comment by insidegui** [comment]
Probably some quirk with the beta. I'll not merge right now because I do want people to be able to use the app reasonably well on El Capitan. I will try to fix this.

**Comment by Tantalum73** [comment]
Ok.
I would love to know if you find the solution or other problems.

File: WWDC/PreferencesWindowController.xib [hunk 1]
```diff
@@ -204,6 +204,16 @@
                             <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                         </textFieldCell>
                     </textField>
+                    <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="ulu-ey-feY">
+                        <rect key="frame" x="14" y="218" width="167" height="32"/>
+                        <buttonCell key="cell" type="push" title="Downlaod All Videos" bezelStyle="rounded" alignment="center" borderStyle="border" imageScaling="proportionallyDown" inset="2" id="sFQ-Gp-P6E">
```
Comment (KurtWagner) [inline WWDC/PreferencesWindowController.xib]: Typo :)

Comment (Tantalum73) [inline WWDC/PreferencesWindowController.xib]: Typo is fixed 👍

**Comment by Tantalum73** [comment]
Have you already tried it with the second β of El Capitan?

**Comment by Tantalum73** [comment]
Or the third?
Is anybody here? 👋🏻

**Pull request closed without merging**